### DISCUSSION TITLE
[nix]: adding aarch64-linux to cachix

### DISCUSSION THREAD

**Discussion by Ladas552** [helix-editor/helix#12341] [Ideas]
### Context
I use `flake.nix` of Helix as an overlay to use master branch of Helix with latest commit on my Nix machines. 
Having 2 systems for hosts: x86_64-linux on desktop and aarch64-linux on Phone with [nix-on-droid](https://github.com/nix-community/nix-on-droid)
### Problem
I use `nix.substituters` to not build Helix dependencies, It works for Desktop, but not for Phone. 
Would be nice to not build Helix for phone. On my system(that uses remote builder with ARM emulation on desktop) it takes 4 hours.
### Extra Info
Basically what is mentioned in [discussion](https://github.com/helix-editor/helix/discussions/10478#discussioncomment-9129501).

Right now, a workaround is pinning helix url input to the latest commit at the time for a month or two, to reduce the amount of time spend building the project.

**Comment by the-mikedavis** [comment]
I thought that there was a prior discussion for adding macOS but I can't seem to find it. The downside with adding aarch64-linux is the same: the cachix storage space is pretty limited (5GB) and we use it up very quickly. Adding other targets would use up more space and could push some artifacts out of the cache for x86 linux.

I would recommend setting up github actions to run on a fork and publish to a separate cache using our cachix workflow as a reference
